src/Input/Silk.NET.Input.Sdl/SdlMouse.cs

```diff
@@ -37,7 +38,7 @@ public SdlMouse(SdlInputContext ctx)
         public override IReadOnlyList<MouseButton> SupportedButtons { get; } = new[]
             {MouseButton.Left, MouseButton.Middle, MouseButton.Right, MouseButton.Button4, MouseButton.Button5};
 
-        public override IReadOnlyList<ScrollWheel> ScrollWheels { get; } = new ScrollWheel[1];
+        public override IReadOnlyList<ScrollWheel> ScrollWheels => _scrollWheels;
```

Comment:
Would rather not unnecessarily introduce a field.

Reply:
The reason I made the `ScrollWheels` backing field explicit was to improve readability and prevent three casts to get back at the scroll wheel data from `IReadOnlyList<ScrollWheel>`. Wouldn't that help?

Reply:
Oh I see, eh fine by me then.